docs/AdvancedConfiguration.md

```diff
@@ -167,36 +167,51 @@ val columnStats =
      |"date_max":"${formatter.format(maxTimestamp)}" }""".stripMargin
 ```
 
-## String indexing via Histograms
+## Indexing via Quantile Based CDF
```

Comment:
Maybe we should also explicitly state that the feature is still experimental.

Reply:
done 👍